1_clusters/binary-auth.tf

```diff
@@ -84,7 +84,9 @@ module "project-services" {
 
 # Create Quality Assurance attestor
 module "quality-attestor" {
-  source = "terraform-google-modules/kubernetes-engine/google//modules/binary-authorization"
+  source = "github.com/terraform-google-modules/terraform-google-kubernetes-engine/modules/binary-authorization"
+  #source = "terraform-google-modules/kubernetes-engine/google//modules/binary-authorization"
```

Comment:
@bgood do you need these commented out lines?  Could you remove them and use the git-history if you want to look at past values?